README.md

```diff
@@ -17,34 +17,27 @@ Add metrics-play dependency in your `build.sbt`:
 
 ```scala
 libraryDependencies += Seq(
-  "org.zalando" %% "markscheider" % "2.5.2"
+  "org.zalando" %% "markscheider" % "2.7.0"
```

Comment:
2.7.0 is already released. I just forgot to open a PR ☺️

```suggestion
  "org.zalando" %% "markscheider" % "2.7.1"
```

Please could you also adjust the version.sbt?

Reply:
np, updated to 2.7.1 - the version in version.sbt, too.